rust/src/data/ffi.rs

```diff
@@ -175,11 +177,45 @@ pub extern "C" fn opendp_data__slice_as_object(
         raw_to_concrete_series(raw).map(AnyObject::new)
     }
 
+    #[cfg(feature = "polars")]
+    pub fn raw_to_dataframe(
+        raw: &FfiSlice
+    ) -> Fallible<AnyObject> {
+        let slices = unsafe { slice::from_raw_parts(raw.ptr as *const *const FfiSlice, raw.len) };
+        let series = slices.iter().map(|&s| raw_to_concrete_series(try_as_ref!(s)))
+        .collect::<Fallible<Vec<Series>>>()?;
+        
+        Ok(AnyObject::new(DataFrame::new(series)?))
+    }
 
+    #[cfg(feature = "polars")]
+    pub fn deserialize_raw<T>(
+        raw: &FfiSlice, name: &str
+    ) -> Fallible<T> where for<'de> T: Deserialize<'de> {
+        let slice = unsafe { slice::from_raw_parts(raw.ptr as *const u8, raw.len) };
+        // https://github.com/pola-rs/pyo3-polars/blob/5150d4ca27c287ff4be5cafef243d9a878a8879d/pyo3-polars/src/lib.rs#L147-L153
```

Comment:
What does this URL do for us? It looks similar to what we're doing, so even if we're just citing sources, wouldn't hurt to make that explicit:
```suggestion
        // Error checking based on pyo3-polars:
        // https://github.com/pola-rs/pyo3-polars/blob/5150d4ca27c287ff4be5cafef243d9a878a8879d/pyo3-polars/src/lib.rs#L147-L153
```